spec/class_author_spec.rb

```diff
@@ -0,0 +1,62 @@
+require_relative '../class_authors_handler'
+require_relative '../class_author'
+
+RSpec.describe AuthorHandler do
+  let(:author_handler) { AuthorHandler.new }
```

Comment:
- [ ] Please make sure to add a test for the methods `can_be_archived?()` and `move_to_archived()` in the item class. :pray:

Reply:
Thank you. I added class_item_spec.rb